yt_dlp/extractor/aiyifan.py

```diff
@@ -0,0 +1,109 @@
+import hashlib
+import re
+import time
+
+from yt_dlp.extractor.common import InfoExtractor
```

Comment:
use relative imports in extractors
```suggestion
from .common import InfoExtractor
```